docs/docs/index.md

```diff
@@ -9,48 +9,48 @@ Welcome to zamba's documentation!
 *Zamba means "forest" in the Lingala language.*
```

Comment:
What is the div at the top supposed to be showing? Looks like it might need a tweak